bitnami/nats/Chart.yaml

```
@@ -31,4 +31,4 @@ maintainers:
 name: nats
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/nats
-version: 8.2.16
+version: 8.2.17
```

Bump a minor since new values were added.
```suggestion
version: 8.3.0
```